std_msgs/Header Header
bool[] state
